broker-cli/broker-daemon-client/index.js

```diff
@@ -58,6 +59,19 @@ const GRPC_STREAM_OPTIONS = {
   'grpc.http2.max_pings_without_data': 0
 }
 
+/**
+ * gRPC client interceptor that allows us to set a deadline for all calls in a
+ * service
+ *
+ * @param {Object} options
+ * @param {Function} nextCall
+ * @returns {grpc.InterceptingCall}
+ */
+function grpcInterceptor (options, nextCall) {
+  options.deadline = grpcDeadline()
```

Comment:
I would only set the deadline if it's not already set, that way you can still specify a longer / shorter deadline if it's desired for a particular call.